sim/team-validator.ts

```diff
@@ -1286,6 +1286,10 @@ export class TeamValidator {
 			}
 		}
 
+		if (dex.species.get(species.baseSpecies).tags?.length > 0) {
```

Comment:
```suggestion
		if (dex.species.get(species.baseSpecies).tags?.length) {
```
0 is falsy so if the length === 0 the if statement wont go through